hack/common.sh

```diff
@@ -1001,6 +1001,23 @@ function os::build::find-binary() {
 }
 readonly -f os::build::find-binary
 
+# os::build::get-go-bin-location-from-gopath locates a go install binary in GOPATH directories 
+# and returns the first path where the binary was found. 
+function os::build::get-go-bin-location-from-gopath() {
+  local matching_gopath=""
+
+  gopaths=`echo $GOPATH | sed 's/:/ /g'`
+  for gopath in $gopaths
+  do
+    if [[ -f "$gopath/bin/$1" ]]; then
```

Comment:
We should make sure to test also that the file we're returning a path to is executable:
```sh
if [[ -f "${part}/bin/${binary_name}" && -x "${part}/bin/${binary_name}" ]]; then
    # return this path
fi
```